consensus/spos/bls/v2/subroundEndRound.go

```diff
@@ -332,38 +342,41 @@ func (sr *subroundEndRound) finalizeConfirmedBlock() bool {
 	return true
 }
 
-func (sr *subroundEndRound) sendProof() {
+func (sr *subroundEndRound) sendProof() (bool, bool) {
```

Comment:
could this be simplified by returning just an error and handling the error in the caller? 

line 268:
`if shouldWaitForMoreSignatures`  =>` if errors.Is(err, spos.ErrInvalidNumSigShares)`

line 272:
`if proofSent` => `if err == nil `

Reply:
ok